File: 09-DesignPatterns-Java/Starter/Strategy/src/RubberDuck.java
public final class RubberDuck extends Duck {
    @Override
    public void quack() {
        System.out.println("Squeak squeak!");
    }

    @Override
    public void display() {
        System.out.println("All yellow, made from rubber.");
    }

    @Override
    public  void fly() {
        System.out.println("Can not fly, not real.");
    }
}

File: 10-DesignPatterns-CSharp/Starter/Strategy/RubberDuck.cs
namespace Strategy
{
    using System;

    public sealed class RubberDuck : Duck
    {
        public override void Quack() => Console.WriteLine("Squeak squeak!");

        public override void Display() => Console.WriteLine("All yellow, made from rubber.");

        public override void Fly() => Console.WriteLine("Can not fly, not real.");
    }
}